src/main/java/com/cx/restclient/ast/dto/common/RemoteRepositoryInfo.java

```diff
@@ -19,11 +19,9 @@
 
     private String branch;
 
-    /**
-     * If access token is used instead of username/password, pass the token into this field.
-     * TODO: add a dedicated field for token.
-     */
     private String username;
 
     private String password;
```

Comment:
Looks like 'password' is no longer needed, because credential type is now always apiKey.